### DISCUSSION TITLE
Add detection do port 631/udp

### DISCUSSION THREAD

**Discussion by paulojjs** [telekom-security/tpotce#1681] [General]
What would be the best way to add detection to any of the existing pots for scans on port 631/udp (because of the CUPS security issue)?

**Comment by t3chn0m4g3** [comment]
While T-Pot has Ipphoney included this is tailored for `tcp/631` instead of `udp/631`.
There is currently no honeypot included that could mimic the desired, vulnerable behaviour necessary to identify attackers.

@referefref has created [CUPS Honeypot](https://github.com/referefref/cupspot-2024-47177) which might be a good fit for T-Pot.
